### PR TITLE
Add function persp-update-names-cache.

### PR DESCRIPTION
**Pull request by roife**
Add function persp-update-persp-names-cache. As mentioned in #143.

### CODE REVIEW

File: persp-mode.el [hunk 1]
```diff
@@ -2253,6 +2261,13 @@ killed, but just removed from a perspective(s)."
 
 ;; Perspective funcs:
 
+(defun persp-update-persp-names-cache (new-persp-names)
```
Comment (Bad-ptr) [inline persp-mode.el:2264]: _**persp**-update-**persp**-names-cache_
In my mind _persp-_ must be used as a prefix only.

Comment (roife) [reply persp-mode.el:2264]: You are right. How about renaming it to `persp-update-names-cache`?

Comment (Bad-ptr) [reply persp-mode.el:2264]: > persp-update-names-cache

Yes, that's it.

Comment (roife) [reply persp-mode.el]: I have renamed it. Do you have any other suggestions?

Comment (Bad-ptr) [reply persp-mode.el]: Old name remains in the commit message.

**Comment by Bad-ptr** [comment]
Thanks

**Pull request merged**